### ISSUE TITLE
Support for decorative images

### ISSUE THREAD

**Issue by kslstn** [rbuchberger/jekyll_picture_tag#157]
I'm trying to follow the [accessibility recommendations for decorative images](https://www.w3.org/WAI/tutorials/images/decorative/). Based on that, for images that should be ignored by screen readers, I want to add the attributes `role="presentation"` and `alt=""`. I've noticed that the `alt` attribute is removed when the alt text of the include is intentionally kept empty like this:
`--alt {{ page.banner-image-alt-text | default:  }}` or
`--alt {{ page.banner-image-alt-text | default: "" }}`

I managed to get `alt=""` to show up by adding this to my `markup_presets` in `picture.yml`:
`img: 'alt=" " role="presentation"`

With this line, the alt attribute doesn't show up:
`img: 'alt="" role="presentation"`

The difference is a space in the alt text. Although adding the space solves my problem for now, I see two issues:
1. The alt text is set empty in the preset and thus can't be added for instances where it would be useful.
2. The ARIA role can't be set on an instance level either.
3. It feels wrong to rely on the space to be removed—I'd much rather just use `alt=""`

A workaround for 1 and 2 would be adding additional presets of course. That said, it would be awesome if you can take a look at this and make pictures more accessible!

**Comment by rbuchberger** [comment]
I learned something today, thank you! It had never occurred to me that anyone would want to set empty attributes.

I'll look into this and get it fixed, though I may not manage it until after the holidays.

**Comment by mutil** [comment]
I couldn't get this to work even with a space in `alt`, it just gets stripped.

I agree that this is a major accessibility feature, needed especially for background images.
